doc/source/docs/user_guide/mergingdata.rst

```diff
@@ -132,3 +139,21 @@ The `how` argument specifies the type of join that will occur and which geometry
 Note more complicated spatial relationships can be studied by combining geometric operations with spatial join.
 To find all polygons within a given distance of a point, for example, one can first use the :meth:`~geopandas.GeoSeries.buffer` method to expand each
 point into a circle of appropriate radius, then intersect those buffered circles with the polygons in question.
+
+Nearest Joins
+~~~~~~~~~~~~~
+
+Proximity based joins can be done via ``sjoin_nearest()``.
+
+``sjoin_nearest()`` shares the ``how`` argument with ``sjoin()``, however it does not support an ``op`` argument since no binary predicates are applied.
```

Comment:
```suggestion
``sjoin_nearest()`` shares the ``how`` argument with ``sjoin()``.
```

I don't think you need to note `op` / `predicates` here since there isn't an expectation that they'd be supported.